centrallix/tests/test_exp_print_00.to

```diff
@@ -0,0 +1,9 @@
+##NAME Query Print function
+# The print function does not chowup in the cmp, so this largly tests for crashes
+query select convert(integer, 12345.54321); print convert(integer, 12345.54321)
+query select convert(double, 12345.54321); print convert(double, 12345.54321)
+query select convert(money, 12345.54321); print convert(money, 12345.54321)
+query select convert(datetime, '1999-02-01'); print convert(datetime, '1999-02-01')
+query select convert(string, '1999-02-01'); print convert(string, '1999-02-01')
+# cannot convert to binary, so use a function that returns the type
+query select from_hex('6f5f4f'); print from_hex('6f5f4f')
```

Comment:
It'd be good to have the "print" output go to the .cmp / .out files as well.  Could you also add a mq function such as mqRedirectPrint() or perhaps objRedirectPrint() that can be used in conjunction with test_obj.c "output" command to redirect that "print" command printf output to a pFile?  Of course fdPrintf() will need to be used instead of printf().  Then if RedirectPrint() is called with a NULL pFile, it can revert to stdout printing.